Description: Remove deprecated Encoding key from desktop file
 The Encoding key is deprecated and all strings must be encoded in UTF-8.
 .
 http://standards.freedesktop.org/desktop-entry-spec/1.0/apc.html
Author: Mark Purcell <msp@debian.org>
Forwarded: https://github.com/LubosD/twinkle/pull/37
Last-Update: 2008-10-11
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/twinkle.desktop.in
+++ b/twinkle.desktop.in
@@ -1,5 +1,4 @@
 [Desktop Entry]
-Encoding=UTF-8
 Name=Twinkle
 GenericName=A SIP softphone
 Comment=A SIP softphone
